client/components/LeftDiv.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import axios from 'axios';
+import ReviewTable from './ReviewTable.jsx';
+
+
+class LeftDiv extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      reviews: [],
+    };
+  }
+
+  componentDidMount() {
+    this.getReviewsByProdId();
+  }
+
+  // get reviews by product id
+
+  getReviewsByProdId() {
```

Comment:
again you do this in your app component so you could just pass this down as a prop to the component